src/internal/internalAPIs.ts

```diff
@@ -232,9 +232,13 @@ export function handleParentMessage(evt: DOMMessageEvent): void {
     const callback = GlobalVars.callbacks[message.id];
     if (callback) {
       callback.apply(null, message.args);
-
-      // Remove the callback to ensure that the callback is called only once and to free up memory.
-      delete GlobalVars.callbacks[message.id];
+      if (
+        !('isPartialResponse' in evt.data) ||
+        !(typeof evt.data.isPartialResponse === 'boolean') ||
```

Comment:
I think it would be clearer if you simply used `!==` instead of `===` and avoided negating the whole expression